script/proto_generate.sh

```diff
@@ -0,0 +1,12 @@
+GIT_IS_AVAILABLE=$(git --version 2>&1 >/dev/null)
+PROTOC_IS_AVAILABLE=$(protoc --version 2>&1 >/dev/null)
+if  $GIT_IS_AVAILABLE && $PROTOC_IS_AVAILABLE 
+then
+	cd ..
+	git clone https://github.com/open-telemetry/opentelemetry-proto
```

Comment:
This will work the first time, but if you attempt to clone a repo that is already cloned, it won't take any action.  We either need to remove this directory at the end of this script, or make changes accordingly.  What do you think we should do here?

Reply:
According to me there should be two possible ways :
1)Delete the folder if there is any existing opentelemetry-proto or proto folder and create new one
2)or to just skip if there is already exist that folder

Reply:
Skipping if the folder exists could be a mistake - an update to the protoc upstream wouldn't translate well in this scenario.